test/Telegram.Bot.Tests.Unit/Serialization/MenuButtonSerializationTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Newtonsoft.Json;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+using Xunit;
+
+namespace Telegram.Bot.Tests.Unit.Serialization;
+
+public class MenuButtonSerializationTests
+{
+    [Fact]
+    public void Should_Deserialize_Menu_Button_Web_App()
+    {
+        var button = new
+        {
+            type = MenuButtonType.WebApp,
+            text = "Test text",
+            web_app = new
+            {
+                url = "https://example.com/link/to/web/app"
+            }
+        };
+
+        string? menuButtonJson = JsonConvert.SerializeObject(button, Formatting.Indented);
+        MenuButton? menuButton = JsonConvert.DeserializeObject<MenuButton>(menuButtonJson);
+
+        MenuButtonWebApp webAppButton = Assert.IsType<MenuButtonWebApp>(menuButton);
+
+        Assert.Equal(MenuButtonType.WebApp, menuButton.Type);
+        Assert.NotNull(webAppButton.Text);
+        Assert.Equal("Test text", webAppButton.Text);
+        Assert.NotNull(webAppButton.WebApp);
+        Assert.NotNull(webAppButton.WebApp.Url);
+        Assert.Equal("https://example.com/link/to/web/app", webAppButton.WebApp.Url);
+    }
+
+    [Fact]
+    public void Should_Serialize_Menu_Button_Web_App()
+    {
+        MenuButtonWebApp webAppButton = new MenuButtonWebApp()
+        {
+            WebApp = new WebAppInfo()
+            {
+                Url = "https://example.com/link/to/web/app"
+            },
+            Text = "Test text"
+        };
+
+        string? webAppButtonJson = JsonConvert.SerializeObject(webAppButton);
+        Assert.Contains(@"""type"":""web_app""", webAppButtonJson);
```

Comment:
I wish we could use [raw-string-literals](
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-11#raw-string-literals)